Data-Pro-Test/Views/Recipes/Index.html

```diff
@@ -0,0 +1,16 @@
+﻿<div>
```

Comment:
Why 2 index pages?
Data-Pro-Test/Views/Recipes/Index.html and Data-Pro-Test/Views/Recipes/Index.cshtml?